diff --git a/python/setup.cfg b/python/setup.cfg
index d71562a..40cdc66 100644
--- a/python/setup.cfg
+++ b/python/setup.cfg
@@ -26,8 +26,6 @@ classifiers =
 
 [options]
 packages = find:
-setup_requires =
-    pkgconfig
 install_requires =
     cffi
     numpy
diff --git a/python/setup.py b/python/setup.py
index 6362473..97572f9 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -17,6 +17,5 @@
 from setuptools import setup
 
 setup(
-    cffi_modules=["build.py:ffibuilder"],
     package_data={"dftd3": ["_libdftd3*.so"]},
 )
